File: c/src/common/tofusoup.c

#include "tofusoup.h"

soup_t * tofusoup_new()
{
	soup_t * soup = soup_new();
	soup->name = "Tofu Soup";
	arraylist_string_add( soup->ingredients,"1 Pound tofu");
	arraylist_string_add( soup->ingredients,"1 cup carrot juice");    
	arraylist_string_add( soup->ingredients,"1/4 cup spirolena");         
	return soup;
}

File: php/src/common/TofuSoup.php
<?

require_once 'Soup.php';


class TofuSoup extends Soup
{

	function __construct()
	{
      	$this->soupName = "Tofu Soup";
      	$this->soupIngredients = [
			  "1 Pound tofu",
			  "1 cup carrot juice",
			  "1/4 cup spirolena",
			];
    }
}
